packages/pyright-internal/src/analyzer/parseTreeUtils.ts

```diff
@@ -277,7 +277,7 @@ export function printExpression(node: ExpressionNode, flags = PrintExpressionFla
             let escapedString = node.token.escapedValue;
             if ((flags & PrintExpressionFlags.DoNotLimitStringLength) === 0) {
                 const maxStringLength = 32;
-                escapedString = escapedString.substring(0, maxStringLength);
```

Comment:
These `slices` aren't really necessary, it's just that `substring` has been deprecated. I figured I'd switch on the files I was looking at. I can put these back if we want to not bother with these changes.